src/api/spec/cassettes/Comments_with_diff/reply_comment/when_under_the_beta_program/displays_the_changes_after_adding_a_new_comment.yml

```diff
@@ -992,27 +1026,27 @@ http_interactions:
       Connection:
       - close
       Content-Length:
-      - '404'
+      - '406'
     body:
       encoding: UTF-8
       string: |
-        <directory name="package_a" rev="6" vrev="6" srcmd5="a09afa482a997030612d990e3b7733aa">
-          <entry name="README.txt" md5="2e6518277c9d443c0ae45a18ec0c0f1d" size="62" mtime="1729497327"/>
-          <entry name="package_a.changes" md5="3bf7f645ebf1daed37ac2524846a9e7c" size="14" mtime="1729496885"/>
-          <entry name="package_a.spec" md5="674078c9d96283bf095d60ffe47009a0" size="470" mtime="1729496885"/>
+        <directory name="package_a" rev="21" vrev="21" srcmd5="fafdb3cebde7662ca252456e669f6005">
+          <entry name="README.txt" md5="08c4f5154f4ac71662adaff8cbbea780" size="63" mtime="1729504347"/>
+          <entry name="package_a.changes" md5="3bf7f645ebf1daed37ac2524846a9e7c" size="14" mtime="1726738320"/>
+          <entry name="package_a.spec" md5="674078c9d96283bf095d60ffe47009a0" size="470" mtime="1726738320"/>
         </directory>
-  recorded_at: Mon, 21 Oct 2024 07:55:34 GMT
+  recorded_at: Mon, 21 Oct 2024 09:52:37 GMT
 - request:
     method: post
-    uri: http://backend:5352/source/source_project/package_a?cmd=diff&expand=1&filelimit=10000&opackage=target_package&oproject=target_project&tarlimit=10000&view=xml&withissues=1
+    uri: http://backend:5352/source/source_project/package_a?cacheonly=1&cmd=diff&expand=1&filelimit=10000&opackage=target_package&oproject=target_project&tarlimit=10000&view=xml&withissues=1
```

Comment:
Why is this changing?

Reply:
TBH I don't know. Let me take a look

Reply:
I removed the cassette, I couldn't make it fail locally, let's see if the CI fails now or It was all in my head...

Reply:
Strange, how everything passes without that cassette... Anyway. It's off the PR now.